[2024-10-16T17:42:29.870+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_binance_dag1.SUI manual__2024-10-15T22:16:56.420534+00:00 [queued]>
[2024-10-16T17:42:29.894+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_binance_dag1.SUI manual__2024-10-15T22:16:56.420534+00:00 [queued]>
[2024-10-16T17:42:29.895+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2024-10-16T17:42:29.928+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): SUI> on 2024-10-15 22:16:56.420534+00:00
[2024-10-16T17:42:29.938+0000] {standard_task_runner.py:57} INFO - Started process 282 to run task
[2024-10-16T17:42:29.942+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'api_binance_dag1', 'SUI', 'manual__2024-10-15T22:16:56.420534+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/binance.Dag1.py', '--cfg-path', '/tmp/tmpr80sx461']
[2024-10-16T17:42:29.946+0000] {standard_task_runner.py:85} INFO - Job 124: Subtask SUI
[2024-10-16T17:42:29.968+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-10-16T17:42:30.028+0000] {task_command.py:410} INFO - Running <TaskInstance: api_binance_dag1.SUI manual__2024-10-15T22:16:56.420534+00:00 [running]> on host 3e776c2a96a1
[2024-10-16T17:42:30.530+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Danil' AIRFLOW_CTX_DAG_ID='api_binance_dag1' AIRFLOW_CTX_TASK_ID='SUI' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T22:16:56.420534+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T22:16:56.420534+00:00'
[2024-10-16T17:42:31.189+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2024-10-16T17:42:31.224+0000] {logging_mixin.py:149} INFO - Pushed to XCom: Key: SUIU, Value: [[1728259200000, '1.84090000', '2.16800000', '1.83640000', '2.01730000', '191514458.70000000', 1728345599999, '385497166.31088000', 937214, '97872860.70000000', '197125781.07188000', '0'], [1728345600000, '2.01730000', '2.13470000', '1.92950000', '1.98680000', '147218821.70000000', 1728431999999, '297046894.50953000', 714963, '76994797.00000000', '155339375.20641000', '0'], [1728432000000, '1.98690000', '1.99740000', '1.84350000', '1.87550000', '118288209.70000000', 1728518399999, '227579008.21677000', 549488, '59190482.90000000', '113868134.87768000', '0'], [1728518400000, '1.87560000', '1.90490000', '1.74670000', '1.83450000', '109114892.50000000', 1728604799999, '198903082.98708000', 526100, '53858444.50000000', '98220848.23254000', '0'], [1728604800000, '1.83470000', '2.08760000', '1.79000000', '2.05400000', '132738620.20000000', 1728691199999, '260806572.95123000', 642281, '68410657.10000000', '134572128.93377000', '0'], [1728691200000, '2.05400000', '2.28820000', '2.00320000', '2.24150000', '130700373.50000000', 1728777599999, '282217223.15618000', 626545, '70457852.40000000', '152187562.12109000', '0'], [1728777600000, '2.24200000', '2.36800000', '2.12350000', '2.34380000', '115307630.40000000', 1728863999999, '257246254.55338000', 601981, '63935745.30000000', '142780521.81575000', '0']]
[2024-10-16T17:42:31.231+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-10-16T17:42:31.237+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2024-10-16T17:42:31.245+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 282. PIDs of all processes in the group: [282]
